roles/openldap/defaults/main.yml

```diff
@@ -0,0 +1,194 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# docker
+
+docker_registry_openldap: quay.io
+
+##########################
+# openldap
+
+openldap_configuration_directory: /opt/openldap/configuration
+openldap_secrets_directory: /opt/openldap/secrets
+openldap_docker_compose_directory: /opt/openldap
+
+openldap_network: 172.31.100.32/28 # TODO: Find a proper one not shared with phpmyadmin
```

Comment:
Check http://docs.osism.de/appendix/tables/service-networks.html. There is a table with all used network. The last assigned one is ``172.31.100.224/28``. The next free one is ``172.31.100.240/28``.

Reply:
Done